src/desdemona/tasks/kafka.clj

```diff
@@ -2,30 +2,12 @@
   (:require [taoensso.timbre :refer [info]]
             [cheshire.core :as json]))
 
-(defn deserialize-message-json [bytes]
```

Comment:
We're not _currently_ using this but we're going to be using it again basically immediately when we want to start parsing the JSON we're getting from syslog-ng/kafka.